docs/zone-pivot-groups.yml

```diff
@@ -10,6 +10,16 @@ groups:
     title: Linux
   - id: os-macos
     title: macOS
+- id: operating-systems-set-two
+  title: Linux Operating System
+  prompt: Choose a Linux distribution
+  pivots:
+  - id: os-linux-redhat
+    title: 'Fedora, CentoS, or RHEL'
```

Comment:
```suggestion
    title: 'Fedora, CentOS, or RHEL'
```

Reply:
Should we add Rocky, Alma, Oracle and other rebuilds here too?

Reply:
@omajid I don't know, my gut says no because we don't call those out in docs at all. They're not listed as supported OS in the dotnet wiki.